mne_connectivity/base.py

```diff
@@ -685,6 +686,10 @@ def get_data(self, output='compact'):
         if output == 'raveled':
             data = self._data
         else:
+            if self.method in ['mic', 'mim', 'gc', 'gc_tr']:
+                raise ValueError('cannot return multivariate connectivity '
```

Comment:
So at the moment it's because the results for a single connection would correspond to multiple entries in the matrix (for the various channels involved), and if we extend the functionality to compute multiple connections in a single function call, we could end up with a case where the results for multiple connections correspond to the same matrix entries (i.e. results for 0 -> 1 would overlap with results for 0 -> [1, 2]). I have added a comment stating this.